examples/counter-hot/store/mutations.js

```diff
@@ -1,9 +1,11 @@
 export const increment = state => {
   state.count++
-  state.history.push('increment')
```

Comment:
IMO, we should leave `state.history.push` along with increment / decrement since they looks atomic. When we increment / decrement `count`, we would always want to update the history.

Reply:
I'm actually also in favor of keeping this in the mutation, for the same reasons @ktsn offered. By trying to decouple these, we make it possible to create an invalid application state with time travel debugging.

Reply:
Great catch guys. The idea was to make the actions atomic but atomic mutations in this case is a better approach, especially for time travel debugging.